src/seedu/addressbook/data/person/Address.java

```diff
@@ -16,20 +16,26 @@
 
 
     public final String value;
-    private boolean isPrivate;
+    //private boolean isPrivate;
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.